lib/internal/process/promises.js

```diff
@@ -99,3 +99,12 @@ function setupPromises(scheduleMicrotasks) {
 
   return emitPendingUnhandledRejections;
 }
+
+const savedString = String;
+function safeString(value) {
+  try {
+    return savedString(value);
+  } catch (err) {
+    return '<toString() threw exception>';
```

Comment:
@thefourtheye The final error message already contains the information that it’s about promise rejection reasons, so that would be duplicating a lot. Also, I think V8 reports the same literal string in similar scenarios.